apps/src/templates/studioHomepages/SpecialAnnouncement.jsx

```diff
@@ -22,28 +22,27 @@ export default class SpecialAnnouncement extends Component {
     const headingText = isEnglish
       ? isTeacher
         ? i18n.teacherAnnouncementSpecialWinter2021Heading()
-        : i18n.studentAnnouncementSpecial2020Heading()
+        : i18n.studentAnnouncementSpecial2022HocHeading()
```

Comment:
String: https://github.com/code-dot-org/code-dot-org/blob/dea9dae59da03c7aa9a10b4c713a6ce2278fde0c/apps/i18n/common/en_us.json#L1072

Reply:
just curious of the english-only strings are just while we wait for translations, or will remain different for english through hoc?

Reply:
It'll be the same copy for English and Int'l throughout — I replaced both just in case the old copy showed up for the Int'l version of the banner.